src/test/java/spoon/test/template/testclasses/InvocationTemplate.java

```diff
@@ -15,15 +15,15 @@ void invoke() {
 	
 	@Local
 	public InvocationTemplate(CtTypeReference<?> ifaceType, String methodName) {
-		this.ifaceType = ifaceType;
-		this.methodName = methodName;
+		this._IFace = ifaceType.getSimpleName();
+		this._$method$ = methodName;
 	}
 
 	@Parameter("IFace")
-	CtTypeReference<?> ifaceType;
+	String _IFace;

```

Comment:
and here I see that field name IFace doeas not start lowercase - java conventions - so may be we should realy relax the contracts of this PR? It is not a problem for me.